UI/forms/OBSBasicSettings.ui

```diff
@@ -8418,8 +8429,6 @@
   <tabstop>lowLatencyBuffering</tabstop>
   <tabstop>baseResolution</tabstop>
   <tabstop>outputResolution</tabstop>
-  <tabstop>downscaleFilter</tabstop>
-  <tabstop>fpsType</tabstop>
```

Comment:
Is this intentional? Do we need to re-add these tabstops somewhere?

Reply:
I definitely discarded these removals more than once and I appear to have accidentally still included them